backend/src/main/java/ca/bc/gov/app/dto/client/ClientListDto.java

```diff
@@ -0,0 +1,11 @@
+package ca.bc.gov.app.dto.client;
+
+public record ClientListDto(
+    String clientNumber, 
+    String clientAcronym, 
+    String clientName,
+    String clientType, 
+    String city, 
+    String clientStatus, 
+    Long count) {
```

Comment:
I would recommend removing the count here and using a Pair instead. This is something I have to fix as well on the forest client api